api/OffscreenCanvasRenderingContext2D.json

```diff
@@ -273,8 +274,42 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
+              "version_added": "105"
+            },
+            "firefox_android": "mirror",
+            "ie": {
               "version_added": false
             },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "commit": {
```

Comment:
Note, this was added in FF105. The chrome version isn't certain but it was certainly around this time based on various chromium bugs - and this matches the parent.